patterns/1-initial/introducing-metrics-in-innersource.md

```diff
@@ -89,8 +100,11 @@ Initial
 - Clint
 - Russ Rutledge
 - Tom
+- Jack Yang
 
 ## Acknowledgement
 
 - Georg
 - Bob
+- Aaron Steward
+- Willson Mar
```

Comment:
```suggestion
- Wilson Mar
```

Possibly spelled like this?

Also did you check with both Aaron and Wilson that they would like to be mentioned here?